web/app/dashboard/add-project-dialog/add-project-dialog.component.ts

```diff
@@ -15,12 +15,13 @@ export interface AddProjectDialogConfig {
 
 interface TriggerOption {
   viewValue: string;
-  value: 'commit'|'nightly';
+  value: 'commit' | 'pull_request' | 'nightly';
 }
 
 const TRIGGER_OPTIONS: TriggerOption[] = [
-  {viewValue: 'for every commit and PR', value: 'commit'},
-  {viewValue: 'nightly', value: 'nightly'},
+  { viewValue: 'for every commit', value: 'commit' },
```

Comment:
i think this might break some of the web app tests

Reply:
I fixed the frontend tests ✅.